CRM/Custom/Form/ChangeFieldType.php

```diff
@@ -146,13 +146,14 @@ public function postProcess() {
     $customField = new CRM_Core_DAO_CustomField();
     $customField->id = $this->_id;
     $customField->find(TRUE);
+    $customField->serialize = in_array($dstHtmlType, $mutliValueOps, TRUE);
```

Comment:
For the value of `serialize`, @colemanw uses `CRM_Core_DAO::SERIALIZE_SEPARATOR_BOOKEND` in CRM/Custom/Form/Field.php rather than a simple boolean, but I don't know if that's necessary here? (it ends up being an integer 1 either way)

Reply:
I think using `CRM_Core_DAO::SERIALIZE_SEPARATOR_BOOKEND` may have been more sensible but this should be ok I suppose. Only issue is that if something used the trimmed separator rather than the bookended but I doubt it